src/com/t_oster/liblasercut/drivers/GenericGcodeDriver.java

```diff
@@ -523,7 +523,7 @@ private void writeShutdownCode() throws IOException {
 
   protected void sendLine(String text, Object... parameters) throws IOException
   {
-    out.format(FORMAT_LOCALE, text+LINEEND(), parameters);
+    out.format(FORMAT_LOCALE, text.replace(" ", "")+LINEEND(), parameters);
```

Comment:
Sorry, my bad, definitely meant for that to be in the Grbl driver! Will make this change now.